app/src/main/java/com/example/pricingpal/view/CategoryList.kt

```diff
@@ -49,41 +54,48 @@ fun CategoryCard(category: Category, navController: NavController){
     Card(
         modifier = Modifier
             //Makes the card clickable, and when clicked navigates to the ItemList passing along the category name as a String
-            .clickable(onClick = {navController.navigate(Screen.ItemList.withArgs(category.category))})
+            .clickable(onClick = { navController.navigate(Screen.ItemList.withArgs(category.category)) })
 
             // padding around the card
             .padding(15.dp)
+            .fillMaxWidth()
             .border(
                 // puts a border around the card
-                border = BorderStroke(3.dp, Color(0xFF22577A)),
+                border = BorderStroke(4.dp, color = colorResource(id = R.color.deep_blue)),
                 // shapes the card
                 shape = RectangleShape
             ),
         // puts a shadow under the card to make it pop out
-        elevation = CardDefaults.cardElevation(12.dp),
+        elevation = CardDefaults.cardElevation(8.dp),
     ) {
-        // Allows for the text to be put on top of the card
-        Box(modifier = Modifier
+        // Column to display image and category name on card
+        Column(modifier = Modifier
             // the size of the box
-            .width(700.dp)
-            .height(200.dp)
-            // changes the color of the box
-            .background(Color(0xFF38A3A5)),
-            //allows for the content to be put into the center of the box
-            contentAlignment = Alignment.Center,
+            .background(color = colorResource(id = R.color.pale_blue))
         ){
+            //Here is where the code to display image would be. Current image is a placeholder
+            Image(
```

Comment:
I'm glad you added the content description for the picture, but why does the Image parameter only have the accessories.png and not the other?

Reply:
I didn't want to add code to call all the images, because we're considering moving them to a database. Any changes I make to implement that now are likely to be reversed and replaced, so I just hard-coded the one image in as a placeholder.

Reply:
Okay. Good to know.